pkg/cmdimpl/install.go

```diff
@@ -13,19 +13,24 @@ var appCRD []byte
 
 type InstallParamSet struct {
 	Namespace string
+	DryRun    bool
 }
 
-func Install(params InstallParamSet) ([]byte, error) {
-	kubectlApply := fmt.Sprintf("kubectl apply --namespace=%s -f -", params.Namespace)
-
-	if err := utils.CallCommandForEffectWithInputPipe(kubectlApply, string(appCRD)); err != nil {
-		return []byte(""), wrapError(err, "could not apply wego manifests")
+func Install(params InstallParamSet) error {
+	manifests, err := fluxops.Install(params.Namespace, params.DryRun)
+	if err != nil {
+		return err
```

Comment:
might be helpful for future debugging something like `fmt.Errorf("error on install %s",err)` or something similar.